Formula/pgxnclient.rb

```diff
@@ -0,0 +1,24 @@
+class Pgxnclient < Formula
+  include Language::Python::Virtualenv
+
+  desc "Command-line client for the PostgreSQL Extension Network"
+  homepage "https://pgxn.github.io/pgxnclient/"
+  url "https://github.com/pgxn/pgxnclient/archive/refs/tags/v1.3.1.tar.gz"
+  sha256 "5d711010b53f257c35e8cb2fe8f954567c736d9af528e0da32227fdcaefe4350"
+  license "BSD-3-Clause"
+
+  depends_on "python@3.9"
+
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/6b/34/415834bfdafca3c5f451532e8a8d9ba89a21c9743a0c59fbd0205c7f9426/six-1.15.0.tar.gz"
+    sha256 "30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259"
+  end
```

Comment:
```suggestion
  depends_on "six"
```
There's a `six` formula this can use.

Reply:
I originally did this, but `depends_on "six"` is in 1 formula, while `resource "six"` is in 159, so switched it for consistency.

Reply:
`six` is a newish formula, so we haven't gotten around to switching them yet. But they really should be switched at some point.